src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: all ease 3s;
+  &:hover {
+    transform: scale(1.2);
+  }
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 5px;
+    color: #060b35;
+    line-height: 18px;
+  }
+  &:hover .card__title {
```

Comment:
You don't need to specify the block name when nesting an element inside the block. Use '&__title' instead of '&:hover .card__title'.